sso-kit-starter/sso-kit-starter.iml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
```

Comment:
I suppose this file slipped into the commit by mistake 🙂

Reply:
Yeah, totally 😅